terraform/projects/infra-jump-instance/README.md

```diff
@@ -0,0 +1,56 @@
+## Project: infra-jump-instance
```

Comment:
It appears these docs have been written manually meaning running `make docs` will overwrite them. Instead they can be included at the top in a docblock of the project file and they will be produced using `make docs`.

Reply:
ok